test/cluster/conftest.py

```diff
@@ -265,9 +270,11 @@ async def manager(request: pytest.FixtureRequest,
     await manager_client.stop()  # Stop client session and close driver after each test
     if cluster_status["server_broken"]:
         pytest.fail(
-            f"test case {test_case_name} leave unfinished tasks on Scylla server. Server marked as broken,"
+            f"test case {test_case_name} left unfinished tasks on Scylla server. Server marked as broken,"
             f" server_broken_reason: {cluster_status["message"]}"
         )
+    if found_errors:
+        pytest.fail(found_errors)
```

Comment:
I don't think it's worth showing all found information there. Better to say which error were found and where to find more information regarding these fails.